data/crac-creation/crac-creation-util/src/main/java/com/farao_community/farao/data/crac_creation/util/cgmes/CgmesBranchHelper.java

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright (c) 2022, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+package com.farao_community.farao.data.crac_creation.util.cgmes;
+
+import com.powsybl.iidm.network.*;
+
+import java.util.Objects;
+
+/**
+ * @author Philippe Edwards{@literal <philippe.edwards at rte-france.com>}
+ */
+public class CgmesBranchHelper {
```

Comment:
would i make sense to implement the ElementHelper interface ? or CnecElementHelper interface ?

Reply:
I'm not sure about the CnecElementHelper (because of the inverted field which does not really make sense for cgmes since the elements are just represented by their ID), but I'll make it implement the ElementHelper interface!